### PR TITLE
DOC: Update pandas.DataFrame.tail docstring

### PR DESCRIPTION
**Pull request by thismakessand**
-Add extended summary

Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
###################### Docstring (pandas.DataFrame.tail)  ######################
################################################################################

Return the last `n` rows.

This function returns last `n` rows from the object based on the
underlying index. It is useful for quickly verifying data,
for example, after sorting or appending rows.

Parameters
----------
n : int, default 5
    Number of rows to select.

Returns
-------
type of caller
    The last `n` rows of the caller object.

See Also
--------
pandas.DataFrame.head : The first `n` rows of the caller object.

Examples
--------
>>> df = pd.DataFrame({'animal':['alligator', 'bee', 'falcon', 'lion',
...                    'monkey', 'parrot', 'shark', 'whale', 'zebra']})
>>> df
      animal
0  alligator
1        bee
2     falcon
3       lion
4     monkey
5     parrot
6      shark
7      whale
8      zebra

Viewing the last 5 lines

>>> df.tail()
   animal
4  monkey
5  parrot
6   shark
7   whale
8   zebra

Viewing the last n lines (three in this case)

>>> df.tail(3)
  animal
6  shark
7  whale
8  zebra

################################################################################
################################## Validation ##################################
################################################################################

Docstring for "pandas.DataFrame.tail" correct. :)

```

### CODE REVIEW

**Review by jorisvandenbossche — COMMENTED** [review]
Looks good!

Small comment on the See Also: can you make this "Return first `n` rows." ?

**Comment by jorisvandenbossche** [comment]
And in the examples section in "Viewing the last n lines (three in this case)" -> can you there also add single backticks around "n" ?

File: pandas/core/generic.py [hunk 1]
```diff
@@ -3660,7 +3660,11 @@ def head(self, n=5):
 
     def tail(self, n=5):
         """
-        Return the last n rows.
+        Return the last `n` rows.
+
+        This function returns last `n` rows from the object based on the
```
Comment (jreback) [inline pandas/core/generic.py]: this is not fully correct. it returns the last n rows by position, regardless of the index.

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/core/generic.py (hunk 1 above)
Comment (thismakessand) [reply pandas/core/generic.py]: Yeah, I'll put in a PR for that one

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**